sops.go

```diff
@@ -0,0 +1,259 @@
+package sops
+
+import (
+	"crypto/sha512"
+	"fmt"
+	"go.mozilla.org/sops/aes"
+	"go.mozilla.org/sops/kms"
+	"go.mozilla.org/sops/pgp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const DateFormat = "2006-01-02T15:04:05Z"
```

Comment:
You don't need to redefine this, just use `time.RFC3339` which is an international standard